scripts/alternative-install-host-dependencies.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+### Install all dependencies on Ubuntu 16.04 LTS (Xenial Xerus) for RChain dev environment
+
+
+## Detect if running in docker container - setup using sudo accordingly
```

Comment:
If you're not on docker, where does this command run? Do you want to make a tempdir or something?

Reply:
This script doesn't download the git repo, just preps the box. The file it "curls" down is unloaded in the /tmp/ directory.
I could have it create and cd to a folder but not really a reason for now.

Reply:
You're right. I'll create temp.